contracts/src/libraries/PlonkVerifier.sol

```diff
@@ -269,14 +269,14 @@ library PlonkVerifier {
 
             // public inputs
             // TODO: update these when change PI from dynamic array to fixed length
```

Comment:
Shall we remove this TODO?

Reply:
done in [cf37205](https://github.com/EspressoSystems/espresso-sequencer/pull/1835/commits/cf3720528e22e7a2b74249a1281615818ecb9928)